src/routingmanager/routingmanager.csproj

```diff
@@ -25,9 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="6.4.0" />

```

Comment:
Thanks @Eneuman  for this PR. are you sure removing Autofac package reference doesn't impact DI ? was this tested locally ?

Reply:
@hsubramanianaks 
Autofac is implicit reference from the Autofac.Extensions.DependencyInjection package so it's available to the code :)

Reply:
Thanks for confirming on this.